### PR TITLE
[WIP][Not4Review] Run ipv6 test only

### PR DESCRIPTION
**Pull request by lzhecheng**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note

```

### CODE REVIEW

File: test/e2e/common.go [hunk 1]
```diff
@@ -274,7 +274,7 @@ func EnsureControlPlaneInitialized(ctx context.Context, input clusterctl.ApplyCl
 	}, input.WaitForControlPlaneIntervals...).Should(Succeed(), "API Server was not reachable in time")
 
 	_, hasWindows := cluster.Labels["cni-windows"]
-	if kubeadmControlPlane.Spec.KubeadmConfigSpec.ClusterConfiguration.ControllerManager.ExtraArgs["cloud-provider"] != "azure" {
+	if false {
```
Comment (CecileRobertMichon) [inline test/e2e/common.go:277]: why this change? the ipv6 job needs to install CCM

**Comment by CecileRobertMichon** [comment]
ipv6 job failed but due to different error

/retest

File: test/e2e/common.go (hunk 1 above)
Comment (lzhecheng) [reply test/e2e/common.go:277]: I'd like to check what will happen without CCM since the impacted Pods should exist before CCM is deployed.

**Comment by CecileRobertMichon** [comment]
> Pod can not be created: 

prow issue

/retest
